src/Sarif.FunctionalTests/ConverterTestData/Semmle/results.csv.sarif

```diff
@@ -0,0 +1,1353 @@
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "This field is in a serializable class,  but is not serializable itself because FileFilter is not serializable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/monitor/FileAlterationObserver.java",
+                "region": {
+                  "startLine": 127,
+                  "startColumn": 30,
+                  "endLine": 127,
+                  "endColumn": 39
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "This field is in a serializable class,  but is not serializable itself because FileFilter is not serializable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/filefilter/DelegateFileFilter.java",
+                "region": {
+                  "startLine": 39,
+                  "startColumn": 30,
+                  "endLine": 39,
+                  "endColumn": 39
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "warning",
+          "message": "This field is in a serializable class,  but is not serializable itself because FilenameFilter is not serializable.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/filefilter/DelegateFileFilter.java",
+                "region": {
+                  "startLine": 37,
+                  "startColumn": 34,
+                  "endLine": 37,
+                  "endColumn": 47
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "toUpperCase without locale parameter.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/input/XmlStreamReader.java",
+                "region": {
+                  "startLine": 745,
+                  "startColumn": 32,
+                  "endLine": 745,
+                  "endColumn": 55
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "toUpperCase without locale parameter.",
+          "locations": [
+            {
+              "resultFile": {
+                "uri": "src/main/java/org/apache/commons/io/output/XmlStreamWriter.java",
+                "region": {
+                  "startLine": 172,
+                  "startColumn": 36,
+                  "endLine": 172,
+                  "endColumn": 59
+                }
+              }
+            }
+          ]
+        },
+        {
+          "level": "note",
+          "message": "toUpperCase without locale parameter.",
```

Comment:
I agree. Not actionable by the converter, of course -- the message comes straight from the Semmle file.

---
In reply to: [90077039](https://github.com/Microsoft/sarif-sdk/pull/545#discussion_r90077039) [](ancestors = 90077039)